docs/DeveloperGuide.md

```diff
@@ -720,40 +720,212 @@ testers are expected to do more *exploratory* testing.
+
+### Tagging Person(s) to an Event
+1. Tagging a person to an Event
+    1. Prerequisites: There must be at least 1 event in the event list and 1 person in the person list.
+
+    2. Whenever a valid tagging is done, all events in the current event list are displayed and the tagged event is updated to reflect the new tagged person.
+
+    3. Test case: `tagEvent 1 p/1`<br>
+       Expected: The first person in the current person list is tagged to the first event in the current event list. A success message is shown in the
+       result display, indicating that the name of the first person has been tagged to the event title of the first event.
+
+    4. Test case: `tagEvent 0 p/1`<br>
+       Expected: No person is tagged to an event. Error message is shown in the result display.
+
+    5. Other incorrect tagEvent commands to try: `tagEvent x p/y` (where either x is larger than the size of the current event list or y is
+       larger than the size of the current person list), `TAGEVENT 1 p/1`, `tagEvent 1 p/x` (where x is the index of a person that is already tagged in the first event
+       in the current event list).<br>
+       Expected: Similar to second test case.
+
+2. Tagging multiple persons to an Event
+    1. Prerequisites: There must be at least 1 event in the event list and more than 1 person in the person list.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+    2. Whenever a valid tagging is done, all events in the current event list are displayed and the tagged event is updated to reflect the new tagged persons.
 
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+    3. Test case: `tagEvent 1 p/1 2`<br>
+       Expected: The first and second person in the current person list is tagged to the first event in the current event list. A success message
+       is shown in the result display, indicating that the name of the first and second person has been tagged to the event title of the first event.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+    4. Test case: `tagEvent 0 p/1 2`<br>
+       Expected: No persons are tagged to an event. Error message is shown in the result display.
 
-1. _{ more test cases …​ }_
+    5. Other incorrect tagEvent commands to try: `tagEvent x p/y z` (where either x is larger than the size of the current event list or y or z is
+       larger than the size of the current person list), `TAGEVENT 1 p/1 2`. `tagEvent 1 p/x y` (where either x or y is the index of a person that is already tagged in the
+       first event in the current event list)<br>
```

Comment:
Can also mention the case that the person indexes are duplicated here, it's ok to not mention it though